packages/backend/src/temp/puter_page_loader.js

```diff
@@ -16,68 +16,66 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-const {encode} = require('html-entities');
-const path_ = require('path');
-const fs_ = require('fs');
+const { encode } = require("html-entities");
+const path_ = require("path");
+const fs_ = require("fs");
 
 const generate_puter_page_html = ({
-    env,
+  env,
 
-    manifest,
-    gui_path,
+  manifest,
+  gui_path,
 
+  app_origin,
+  api_origin,
+
+  meta,
+
+  gui_params,
+}) => {
+  const e = encode;
+
+  const { title, description, short_description, company, canonical_url } =
+    meta;
+
+  gui_params = {
+    ...meta,
+    ...gui_params,
     app_origin,
     api_origin,
+    gui_origin: app_origin,
+  };
 
-    meta,
+  const asset_dir = env === "dev" ? "" : "";
```

Comment:
We need the switch between dev using /src and production deploy using /dist

Reply:
With the current server implementation it gives a 404 because the url is src/** instead of /**. When I removed the /src it worked. Maybe someone should look at the server code.

Reply:
I didn't remove the asset_dir declaration because I understood what it meant. In the dev-server implementation there are two middlwwares for serving static assets (./ and ./src or ./dist) that's why it doesn't break.